rest-api-spec/src/main/resources/rest-api-spec/api/security.saml_authenticate.json

```diff
@@ -0,0 +1,27 @@
+{
+  "security.saml_authenticate":{
+    "documentation":{
+      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api-saml-authenticate.html",
+      "description":"Exchanges a SAML Response message for an elasticsearch access token and refresh token pair"
+    },
+    "stability":"stable",
+    "visibility":"public",
+    "headers":{
+      "accept": [ "application/json"]
```

Comment:
```
     "content_type": ["application/json"]
```
I don't know if this is used.

Reply:
@sethmlarson should we remove this? We have it in a number of specs too, should we remove it from all in a follow up?